src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/LineLengthCheckTest.java

```diff
@@ -99,7 +99,7 @@ public void countUnicodePointsOnce() throws Exception {
             "16: " + getCheckMessage(MSG_KEY, 100, 149),
         };
 
-        verify(checkerConfig, getPath("InputLineLengthUnicodeChars.java"), expected);
+        verifyWithInlineConfigParser(getPath("InputLineLengthUnicodeChars.java"), expected);
```

Comment:
Do not need to keep the variable `checkerConfig` in this test?

Reply:
thanks a lot , fixed.